staging/src/k8s.io/apiserver/pkg/admission/plugin/cel/compile_test.go

```diff
@@ -135,6 +140,41 @@ func TestCompileValidatingPolicyExpression(t *testing.T) {
 			},
 			envType: environment.NewExpressions,
 		},
+		{
+			name: "valid namespaceObject",
+			expressions: []string{
```

Comment:
I was also expecting to see tests that check the behavior of this field when the resource is cluster scoped.

Reply:
I guess E2E test would help that.

Reply:
Any test that shows that cluster scoped resources are working with this change would be satisfactory to me.

Reply:
Because of how the tests are implemented right now, the integration test suite tests "what happens when namespace is nil" while the e2e tests can truly test on a cluster-scoped resource.